### PR TITLE
Strip down error data set when sent to viewer

### PR DESCRIPTION
**Pull request by alabiaga**
fixes: #13867 

- minor refactoring of getErrorReportData
- strip down error report data sent to viewer

### CODE REVIEW

**Comment by alabiaga** [comment]
Note will follow up with doc changes PR documenting this behavior.

**Comment by alabiaga** [comment]
@jridgewell Thanks!!!!!!!!!!!!!!

File: build-system/tasks/bundle-size.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ const log = require('fancy-log');
 const {getStdout} = require('../exec');
 
 const runtimeFile = './dist/v0.js';
-const maxSize = '81.22KB';
+const maxSize = '81.26KB';
```
Comment (choumx) [inline build-system/tasks/bundle-size.js]: Might be smaller if you use object destructuring and an extern instead.

File: test/functional/test-error.js [hunk 2]
```diff
@@ -166,10 +167,11 @@ describe('maybeReportErrorToViewer', () => {
         .then(() => expect(sendMessageStub).to.not.have.been.called);
   });
 
-  it('should send viewer message named `error`', () => {
+  it('should send viewer message named `error` with stripped down error data '
+    + 'set', () => {
     return maybeReportErrorToViewer(win, data)
         .then(() => expect(sendMessageStub).to.have.been
-            .calledWith('error', data));
+            .calledWith('error', errorReportingDataForViewer(data)));
```
Comment (choumx) [inline test/functional/test-error.js]: We should explicitly check the passed keys to prevent regressions.

**Pull request merged**